src/v/datalake/protobuf_to_arrow_converter.cc

```diff
@@ -184,4 +186,17 @@ void error_collector::AddWarning(
   int line, int column, const std::string& message) {
     datalake_log.warn("Protobuf Warning {}:{} {}", line, column, message);
 }
+#else
+void error_collector::RecordError(
+  int line, int column, std::string_view message) override {
+    // Warning level because this is an error in the input, not Redpanda
+    // itself.
+    datalake_log.warn("Protobuf Error {}:{} {}", line, column, message);
```

Comment:
nit: vlog?

Reply:
I don't think that's useful, the line and column are coming from the protobuf library already, adding additional constant information is not useful I think.

Reply:
Oh, aren't line and column here referring to line and column in the input protobuf schema, rather than location in code?

https://github.com/protocolbuffers/protobuf/blob/479a80cf4bbb9ac55565d52e9908b3af838d0763/src/google/protobuf/io/tokenizer.h#L61-L63

Reply:
Ah right you are - I will fix to use vlog

Reply:
Done